packages/config/config/devices/0x010f/fgwceu-201.json

```diff
@@ -0,0 +1,145 @@
+{
+	"manufacturer": "Fibargroup",
+	"manufacturerId": "0x010f",
+	"label": "FGWCEU-201",
+	"description": "Walli Controller",
+	"devices": [
+		{
+			"productType": "0x2301",
+			"productId": "0x1000"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"13": {
+			"label": "LED Frame – Brightness",
```

Comment:
To match our files elsewhere, please make this "LED Frame: Brightness". Carry to the others as well.